share/p2p/peers/manager.go

```diff
@@ -39,6 +37,9 @@ const (
 	// eventbusBufSize is the size of the buffered channel to handle
 	// events in libp2p
 	eventbusBufSize = 32
+
+	// amountOfStoredPools is the amount of pools for recent headers that will be stored in the peer manager
+	amountOfStoredPools = 10
```

Comment:
I'm open for better name for this var

Reply:
Not much to suggest. Maybe storedPoolsAmount